### PR TITLE
test: Change parallelism for java tests

### PR DESCRIPTION
**Pull request by grianbrcom**

### CODE REVIEW

**Review by temanbrcom — APPROVED** [review]
🚀

File: server/pom.xml [hunk 1]
```diff
@@ -280,9 +280,7 @@
                 <artifactId>maven-surefire-plugin</artifactId>
                 <version>${maven.surefire.plugin.version}</version>
                 <configuration>
-                    <forkMode>always</forkMode>
-                    <forkCount>3</forkCount>
-                    <reuseForks>true</reuseForks>
+                    <parallel>all</parallel>
```
Comment (asatklichov) [inline server/pom.xml:283]: Any impact on Jenkins memory, during dev-branch build?

**Review by asatklichov — APPROVED** [review]
🏯

File: server/pom.xml (hunk 1 above)
Comment (grianbrcom) [reply server/pom.xml:283]: I think that limit is 2G according this:
```<argLine>${argLine} -Xmx2048m -XX:MaxPermSize=1024m -Djava.awt.headless=true</argLine>```

**Comment by grianbrcom** [comment]
So, 12 minutes instead of 40. It's better, but not good.

**Pull request merged**